containers/buildUbuntu6.docker/tools/gcc

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/sccache gcc "$@"
```

Comment:
\n?

Reply:
Should not harm. Worked fine with older containers.